simhalt.cc

```diff
@@ -2610,6 +2610,46 @@ void haltestelle_t::get_short_freight_info(cbuffer_t & buf) const
 }
 
 
+void haltestelle_t::get_transfers_info(cbuffer_t& buf) const
+{
+	int idx = get_finance_history( 1, HALT_ARRIVED) == 0 ||
+		get_finance_history( 1, HALT_DEPARTED) == 0 ? 0 : 1;
+	auto transfers = get_finance_history( idx, HALT_ARRIVED) <
+		get_finance_history( idx, HALT_DEPARTED) ?
+		get_finance_history( idx, HALT_ARRIVED) :
+		get_finance_history( idx, HALT_DEPARTED);
+	buf.printf("%d ", transfers);
+	buf.append(translator::translate("transfers"));
+	buf.append("\n");
+}
+
+void haltestelle_t::get_percent_info(cbuffer_t& buf) const
+{
+	bool got_one = false;
+	sint64 sum = 0;
+	for(unsigned int i=0; i<goods_manager_t::get_count(); i++) {
+		const goods_desc_t *wtyp = goods_manager_t::get_info(i);
+		if(gibt_ab(wtyp)) {
+			sum += get_capacity( i>2?2:i );
```

Comment:
[Q] Why do you cap the index with 2?

With this calculation, I think the waiting room occupancy will be lower for halts which accept multiple categories of goods. Is it better to count only the waiting passenger, or will you consider all waiting goods?

Reply:
Technically at the moment, all goods are considered since if you notice a halt being bigger than a certain percentage and you want to investigate you might open the halt to look into it.
Altho I can also understand that might want to show all the capacities separately.

Reply:
I guess that is true yeah. As fo where I got the index 2 from that is probably a misunderstanding from me from the goods_manager
![image](https://github.com/user-attachments/assets/39892fcb-ae23-4d7a-a7d8-486b9b71c07f)
While you do have a point I feel like at least mail should also be included. Then again some people might want to know it about other goods too...